packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -30,12 +30,15 @@
 %global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
 %global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d
 
+%global foreman_salt_datadir /usr/share/foreman-salt
```

Comment:
I guess this should be %{_root_datadir}/foreman-salt/

Reply:
Is `foreman-salt` the right directory given this is a smart proxy plugin?